cmd/plume/prerelease.go

```diff
@@ -62,7 +62,19 @@ var (
 	}
 	platformList []string
 
+	systems = map[string]system{
+		"cl": system{
+			displayName: "ContainerLinux",
+			handler:     runCLPreRelease,
+		},
+		"fedora": system{
+			displayName: "Fedora",
+			handler:     runFedoraPreRelease,
```

Comment:
Have you looked at all at https://github.com/coreos/coreos-assembler/ - I have just started looking at the Plume code and I'm still trying to figure out how these intersect.

See https://github.com/coreos/coreos-assembler/issues/162

Are you thinking that we'd add a 3rd entry here for "FedoraCoreOS"?  Or are they close enough that they're the same?  I am not sure about that since currently Fedora content comes in composes, but that won't be true of FCOS...unless we change something there.

Reply:
What I've noticed that the Fedora way of releasing is almost similar to that of CL, I don't think the same is true for FCOS. We need to add a different way to handle the FCOS scenario.

Reply:
Maybe we should have a quick meeting about this?  It'd be useful to have representation from people who understand all 3 release processes and figure out how we intersect them.  The "iron is hot" so to speak for coreos-assembler; we can make breaking changes if we need to.